main.go

```diff
@@ -289,6 +292,79 @@ Arguments:
 	verifyCmd.Flags().StringVar(&serverConfigPathArg, "config-path", "/etc/opk/config.yml", "Path to the server config file. Default: /etc/opk/config.yml.")
 	rootCmd.AddCommand(verifyCmd)
 
+	clientCmd := &cobra.Command{
+		Use:     "client [subcommand]",
+		Short:   "Interact with client configuration",
+		Example: `  opkssh client provider list`,
+		Args:    cobra.ExactArgs(0),
+	}
+
+	providerCmd := &cobra.Command{
+		Use:     "provider [subcommand]",
+		Short:   "Interact with provider configuration",
+		Example: `  opkssh provider list`,
+		Args:    cobra.ExactArgs(0),
+	}
+
+	providerListCmd := &cobra.Command{
+		Use:     "list",
+		Short:   "List configured providers",
+		Example: `  opkssh provider list`,
+		Args:    cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client_config, err := config.GetClientConfigFromFile(configPathArg, afero.NewOsFs())
+
+			if err != nil {
+				log.Fatal("Unable to load providers. ", err)
+			}
+
+			providersMap, err := config.CreateProvidersMap(client_config.Providers)
+
+			if err != nil {
+				log.Fatal("Unable to load providers. ", err)
+
+			}
+
+			t := table.NewWriter()
+			t.SetOutputMirror(os.Stdout)
+			t.AppendHeader(table.Row{"Providers", "Issuer"})
+
+			for alias, p := range providersMap {
+				t.AppendRow(table.Row{alias, p.Issuer})
+			}
+			t.Render()
+
+			return nil
+		},
+	}
+
+	providerListCmd.Flags().StringVar(&configPathArg, "config-path", "", "Path to the client config file. Default: ~/.opk/config.yml on linux and %APPDATA%\\.opk\\config.yml on windows.")
+
+	providerCmd.AddCommand(providerListCmd)
+
+	providerInitCmd := &cobra.Command{
+		Use:     "init",
+		Short:   "Initialize client provider configuration",
+		Example: `  opkssh provider init`,
```

Comment:
Let's rename this to `opkssh client init` or `opkssh client config init` since the client config will eventually include a lot more than just providers

Reply:
moved to `opkssh client init-config`